monai/bundle/workflows.py

```diff
@@ -50,14 +50,16 @@ class BundleWorkflow(ABC):
 
     def __init__(self, workflow: str | None = None):
         if workflow is None:
-            self.properties = None
+            self.properties = copy(MetaProterties)
             self.workflow = None
             return
         if workflow.lower() in self.supported_train_type:
             self.properties = copy(TrainProperties)
+            self.properties.update(copy(MetaProterties))
```

Comment:
No need `copy()` for the update function I think? CC @wyli .

Thanks.

Reply:
Thanks, @Nic-Ma and @ericspod, have updated it according to Eric's suggestion in https://github.com/Project-MONAI/MONAI/pull/6835/commits/63d3a845b92a643773e08a05482b939bbb88e20d.